Documentation/helm-operator.md

```diff
@@ -88,40 +88,57 @@ The command removes all the Kubernetes components associated with the chart and
+| `csi.enableRbdDriver`           | Enable Ceph CSI RBD driver.                                                                             | `true`                                                 |
+| `csi.enableCephfsDriver`        | Enable Ceph CSI CephFS driver.                                                                          | `true`                                                 |
+| `csi.enableGrpcMetrics`         | Enable Ceph CSI GRPC Metrics.                                                                           | `true`                                                 |
+| `csi.provisionerTolerations`    | Array of tolerations in YAML format which will be added to Ceph CSI provisioner deployment.             | <none>                                                 |
+| `csi.provisionerNodeAffinity`   | The node labels for affinity of `rook-agent` (***)                                                                                                        | <none>                                                 |
+| `csi.pluginTolerations`         | Array of tolerations in YAML format which will be added to Ceph CSI provisioner deployment              | <none>                                                 |
+| `csi.pluginNodeAffinity`        | The node labels for affinity of the Ceph CSI RBD plugin DaemonSet (***)                                                                                                        | <none>                                                 |
+| `csi.cephfsGrpcMetricsPort`     | CSI CephFS driver GRPC metrics port.                                                                    | `9091`                                                 |
+| `csi.cephfsLivenessMetricsPort` | CSI CephFS driver metrics port.                                                                         | `9081`                                                 |
+| `csi.rbdGrpcMetricsPort`        | Ceph CSI RBD driver GRPC metrics port.                                                                  | `9090`                                                 |
+| `csi.rbdLivenessMetricsPort`    | Ceph CSI RBD driver metrics port.                                                                       | `8080`                                                 |
+| `csi.kubeletDirPath`            | Kubelet root directory path (if the Kubelet uses a different path for the `--root-dir` flag)            | `/var/lib/kubelet`                                     |
+| `csi.cephcsi.image`             | Ceph CSI image.                                                                                         | `quay.io/cephcsi/cephcsi:v1.2.1`                       |
+| `csi.registrar.image`           | Ceph CSI registrar image.                                                                               | `quay.io/k8scsi/csi-node-driver-registrar:v1.1.0`      |
```

Comment:
`Ceph CSI` need to be replaced with `Kubernetes CSI` for attacher, provisioner,snapshotter, and registrar as this are not specific to ceph-csi